bundles/org.openhab.binding.yeelight/src/main/java/org/openhab/binding/yeelight/internal/lib/enums/DeviceType.java

```diff
@@ -27,6 +27,8 @@ public enum DeviceType {
     ceiling1,
     ceiling3,
     ceiling4,
+    ceil26,
+    ceiling11,
```

Comment:
I suppose the documentation needs to be enhanced.

Reply:
Updated to reflect all supported models.